src/runtime/virtcontainers/kata_agent.go

```diff
@@ -2090,120 +2090,120 @@ func (k *kataAgent) removeStaleVirtiofsShareMounts(ctx context.Context) error {
 	return err
 }
 
-type reqFunc func(context.Context, interface{}) (interface{}, error)
+type reqFunc func(context.Context, interface{}, *kataclient.AgentClient) (interface{}, error)
 
 func (k *kataAgent) installReqFunc(c *kataclient.AgentClient) {
```

Comment:
I was looking at this function `installReqFunc` and looks like it really does not need the parameter passed to it `c *kataclient.AgentClient` at all. While you are at it, maybe you can drop the parameter as well to clean up this function further.

Reply:
@amshinde Good idea! I have fixed it. PTAL.